pkg/kubelet/pod/mirror_client.go

```diff
@@ -73,21 +76,29 @@ func (mc *basicMirrorClient) CreateMirrorPod(pod *v1.Pod) error {
 	return err
 }
 
-func (mc *basicMirrorClient) DeleteMirrorPod(podFullName string) error {
+func (mc *basicMirrorClient) DeleteMirrorPod(podFullName string, uid *types.UID) (bool, error) {
```

Comment:
Better add godoc explaining the bool return value.

Reply:
Done.

Reply:
and UID mismatch are -> or UID mismatch is

since the two conditions may not hold at the same time.

Reply:
Yeah, OK...

Reply:
Fixed.